web_development_101/project_html_css.md

```diff
@@ -77,7 +77,7 @@ Github allows you to publish web projects directly from a github repository.  Do
 There are a couple of ways to go about doing this, but the simplest is this:
 
 - make sure that the main html file of your project is called `index.html`.  If it is not, you will need to rename it.
-- go to your github repo on the web and click the __Settings__ button
+- go to your online github repository for the specific project eg.`https://github.com/YourName/google-homepage` and click the __Settings__ button for that repository.
```

Comment:
Hi @RudiBoshoff, thanks for this PR. Do you think these updated instructions are needed? The extra words make this a little more unreadable to me.

Reply:
I guess not, but I personally did struggle to realize that "github repo" was specifying the repository of the file I was required to upload and not my overall github account